influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -222,7 +231,15 @@ impl WriteBufferImpl {
             last_cache,
             persisted_files,
             buffer: queryable_buffer,
-        })
+        });
+        let write_buffer: Arc<dyn WriteBuffer> = result.clone();
+        let triggers = result.catalog().inner().read().triggers();
```

Comment:
Better to make this a function on `Catalog` rather than going into its internals to get it.